manifests/p/Patchdeck/Agent/2.5.0/Patchdeck.Agent.installer.yaml

```
@@ -0,0 +1,18 @@
+﻿# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Patchdeck.Agent
+PackageVersion: 2.5.0
```

```suggestion
PackageVersion: 2.5.0.0
```
This should match Control Panel